src/components/predictive-layer/predictive-layer.vue

```diff
@@ -1,198 +1,184 @@
 <template>
```

Comment:
I think the remove buttons should be aligned with the clear input button.

![image](https://user-images.githubusercontent.com/75546736/219644267-ac08c1a8-3006-41da-9226-f06e56c5d7a1.png)

Reply:
This is because of the `x-suggestion-group` XDS component and how it styles the button (aspect ratio of 1 and inherits the min height). We fix this here or make changes to the component? (voting for changing the component, maybe a new variable, like we did for the `x-button` for similar problems)

Reply:
fixed